lcd/pyb_i2c_lcd_test.py

```diff
@@ -1,7 +1,7 @@
 """Implements a HD44780 character LCD connected via PCF8574 on I2C."""
 
 from pyb import I2C, delay, millis
-from lcd_api import LcdApi
```

Comment:
I had to add that line on my OpenMV board or I ran into recursion depth issues.

pyb_i2c_lcd_test was importing pyb_i2c_lcd, which was importing lcd_api. On my pyboard this works fine, but on my OpenMV board, it would fail. By adding an import of LcdApi at this level, the import of LcdApi in pyb_i2c_lcd would just import the already imported module.